src/test/tests/rendering/offscreensave.py

```diff
@@ -139,9 +142,10 @@ def GeneralTests():
     slider.Delete()
     DeleteAllPlots()
 
-def TestLargeImage():
-    # Test saving a 16384 x 16384 image
+def TestLargeImage(width, height, md5sum):
+    # Test saving a large image
     OpenDatabase(silo_data_path("curv2d.silo"))
+    DeleteAllPlots()
```

Comment:
May I suggest moving the DeleteAllPlots to the end of the method, or at least to after the SaveWindow.
And move the OpenDatabase to before the first call to TestLargeImage, and add CloseDatabase at the end of the TestLargeImage calls.